### DISCUSSION TITLE
GitHub responded "Git LFS is disabled for this repository"

### DISCUSSION THREAD

**Discussion by tsuchm** [community/community#23937] [New to GitHub]
<p>When I pushed 40MB jsonl file on <a href="https://github.com/tutcsis/DOIW/" rel="noopener nofollow ugc">https://github.com/tutcsis/DOIW/</a>, GitHub responded as follows:</p>
<pre><code class="lang-auto">$ git lfs install
$ cat .gitattributes 
*.jsonl filter=lfs diff=lfs merge=lfs -text
$ git add large.jsonl
$ git commit
$ git push
batch response: Git LFS is disabled for this repository.                                                                                                                     
error: failed to push some refs to 'git@github.com:tutcsis/DOIW.git'
</code></pre>
<p>What should I do?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/tsuchm">@tsuchm</a></p>
<p>We have gone and enabled  LFS again. It’s not clear why it was disabled, but you should be good to go now. Please let me know if you have any further issue with this.</p>

**Comment by tsuchm** [comment]
<p>Thanks!  The problem has been fixed.</p>
